meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -564,9 +564,9 @@ OSTREE_KERNEL_ARGS:sun8i ?= "earlycon console=ttyS0,115200 ${OSTREE_KERNEL_ARGS_
 ## iMX targets should use the u-boot release based on the NXP BSP
 PREFERRED_VERSION_u-boot-fio:imx ?= "2021.04"
 ## Prefer OP-TEE releases from our layer instead of using the .imx fork
-PREFERRED_VERSION_optee-client:mx8 = "3.15.0"
-PREFERRED_VERSION_optee-examples:mx8 = "3.15.0"
-PREFERRED_VERSION_optee-test:mx8 = "3.15.0"
+PREFERRED_VERSION_optee-client:mx8 = "3.17.0"
```

Comment:
The 3.15.0 version its been removed in this series so I think a PREFERRED_VERSION don't make much sense as we only have the version 3.17.0 now

Reply:
These overrides are needed because of https://github.com/Freescale/meta-freescale/blob/master/conf/machine/include/imx-base.inc#L439.

While we use nxp-bsp (override in meta-freescale to say that we want to consume the bsp stuff from nxp, such as drivers and so on), we don't want to use the imx-based revisions, so we need to override them here.

Reply:
Understood, so it is basically to undo their definitions
```
# Use i.MX optee Version
PREFERRED_VERSION_optee-os:mx8-nxp-bsp     ?= "3.15.0.imx"
PREFERRED_VERSION_optee-client:mx8-nxp-bsp ?= "3.15.0.imx"
PREFERRED_VERSION_optee-test:mx8-nxp-bsp   ?= "3.15.0.imx"
```